src/components/addBook.js

```diff
@@ -0,0 +1,16 @@
+const addBook = () => (
```

Comment:
- [ ] I would suggest naming component files and functions in uppercase like add book to AddBook. Because it helps whether the file is a component that is being rendered or doing some functionality, maybe testing or reducing stuff 👋🏼

Reply:
Thanks, @Rustamxon7 
Really the addBook was not working because the first letter was not in capital